docusaurus.config.js

```diff
@@ -80,6 +80,18 @@ const config = {
         'Licensed by ComCODE with <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/" target="_blank" rel="noopener noreferrer">CC BY-NC-SA 4.0</a> for creative and educational use.',
     },
   },
+
+  // Global markdown config
+  // See: https://docusaurus.io/docs/api/docusaurus-config#markdown
+  markdown: {
+    format: "mdx",
+    mermaid: false,
```

Comment:
We don't have any diagrams yet, but I hardly think that that will remain the case.

Reply:
Knowing that it is disabled by default (now explicitly configured), and enabling it is not required for the upgrade, changing this may be out of scope for this PR. I agree that this feature would be great to have in the future, whether it's enabled now or when needed.